torch/csrc/jit/python/init.cpp

```diff
@@ -1846,6 +1846,43 @@ void initJITBindings(PyObject* module) {
                 return nullptr;
               }),
           py::call_guard<py::gil_scoped_release>());
+
+  py::class_<PythonAwaitWrapper, std::shared_ptr<PythonAwaitWrapper>>(
+      m, "Await")
```

Comment:
this is where it would be `Await` -> `_Await` if we were going to do the `torch._C.Await -> torch._C._Await`